content/en/observability_pipelines/guide/run_multiple_pipelines_on_the_same_host.md

```diff
@@ -0,0 +1,126 @@
+
+    ```
+    [Unit]
+    Description="Observability Pipelines Worker"
+    Documentation=https://docs.datadoghq.com/observability_pipelines/
+    After=network-online.target
+    Wants=network-online.target
+
+    [Service]
+    User=observability-pipelines-worker
+    Group=observability-pipelines-worker
+    ExecStart=/usr/bin/observability-pipelines-worker run
+    Restart=always
+    AmbientCapabilities=CAP\_NET\_BIND\_SERVICE
+    EnvironmentFile=-/etc/default/observability-pipelines-worker
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+- An environment file: `/etc/default/observability-pipelines-worker`, which looks like:
+
+    ```
+    DD\_API\_KEY=\<datadog\_api\_key\>
+    DD\_SITE=datadoghq.com
+    DD\_OP\_DESTINATION\_DATADOG\_ARCHIVES\_AWS\_ACCESS\_KEY\_ID=\<redacted\>
+    DD\_OP\_DESTINATION\_DATADOG\_ARCHIVES\_AWS\_SECRET\_ACCESS\_KEY=\<redacted\>
+    DD\_OP\_PIPELINE\_ID=\<pipeline\_id\>
+    DD\_OP\_SOURCE\_DATADOG\_AGENT\_ADDRESS=0.0.0.0:8282
+    DD\_OP\_API\_ENABLED=true
+    ````
+
+- A data directory: `/var/lib/observability-pipelines-worker`
+
+### Configure an additional Worker
+
+In this example, another pipeline was created with the Fluent source. To configure a Worker for the pipeline:
+
+1. Run the following command to create a new data directory, updating `op-fluent` to a directory name that fits your use case:
+    ```
+    sudo mkdir /var/lib/op-fluent
+    ```
+1. Run the following command to change the owner of the data directory to `observability-pipelines-worker:observability-pipelines-worker`. Make sure to update `op-fluent` to your specific directory name.
+    ```
+    sudo chown -R observability-pipelines-worker:observability-pipelines-worker /var/lib/op-fluent/
+    ```
+1. Create an environment file for the new systemd service, such as `/etc/default/op-fluent` where `op-fluent` is replaced with your specific filename. An example of the file content:
+    ```
+    DD_API_KEY=<datadog_api_key>
+    DD_OP_PIPELINE_ID=<pipelines_id>
+    DD_SITE=[datadoghq.com](http://datadoghq.com)
+    DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_ACCESS_KEY_ID=<aws_access_key_id>
+    DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_SECRET_ACCESS_KEY=<aws_secret_access_key>
+    DD_OP_DESTINATION_DATADOG_ARCHIVES_AZURE_BLOB_CONNECTION_STRING=<connection_string>
+    DD_OP_SOURCE_FLUENT_ADDRESS=0.0.0.0:9091
+    DD_OP_DATA_DIR=/var/lib/op-fluent
+    DD_OP_API_ENABLED=true
+    ```
+    In this example:
+    - `DD_OP_DATA_DIR` is set to `/var/lib/op-fluent`, the new data directory you created in the previous step.
+    - `DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_ACCESS_KEY_ID=<redacted>` and `DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_SECRET_ACCESS_KEY=<redacted>` are environment variables for the Datadog Archives Amazon destination. Replace it with the environment variables for your destinations. See [Environment Variables](https://docs.datadoghq.com/observability_pipelines/environment_variables/?tab=sources).
```

Comment:
```suggestion
    - `DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_ACCESS_KEY_ID=<redacted>` and `DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_SECRET_ACCESS_KEY=<redacted>` are environment variables for the Datadog Archives Amazon destination. Replace them with the environment variables for your destinations. See [Environment Variables](https://docs.datadoghq.com/observability_pipelines/environment_variables/?tab=sources).
```

Reply:
Also for the link can you move https://docs.datadoghq.com/observability_pipelines/environment_variables/?tab=sources
to the bottom of the page in the proper link formatting?
i.e.
[1]: /observability_pipelines/environment_variables/?tab=sources

Reply:
All links have been moved down